great-idea/css/index.css

```diff
@@ -63,4 +64,216 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+
+
+
+
+a {
+  text-decoration: none;
+}
+
+
+a:link {
+  color: black;
+}
+
+a:visited {
+  color: grey;
+}
+
+a:hover {
+  border-bottom: 1px solid;
+  background: #CDFEAA;
+}
+
+nav ul li{
+
+  display: inline-block;
+  margin-right: 50px;
+  margin-top: 5px;
+
+}
+
+.container {
+  max-width: 900px;
+    margin: auto;
+    background: white;
+    padding: 10px;
+}
+
+p {
+  margin-bottom: 50px;
+}
+
+h1 {
+  font-size: 50px;
+}
+
+h2 {
+  margin-top: 50px;
+}
+
+
+
+
+
+
+.middle-img {
+
+}
+
+.top{
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+
+.big-text {
+  text-align: center;
+
+
+}
+
+.round-img {
+
+}
```

Comment:
you're going to want to get into the habit of clearing out unused selectors like this before you finish up